addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -452,6 +452,36 @@
 </template>
 
 <!-- ======= Documentation Course content: cards / categories=======  -->
+<template id="course_promoted_slide" name="Documentation Course content: promoted slide">
+    <div t-if="not search and not search_slide_type and slide_promoted" class="container py-1 mb-2 promoted-slide d-none">
```

Comment:
classes are written in underscore notation and prefixed with module name to avoid collision.

`promoted-slide` => `o_wslides_promoted_slides`